ministere-sante/2020-03-19.yaml

```diff
@@ -0,0 +1,18 @@
+date: 2020-03-19
+source:
+  nom: Ministère des Solidarités et de la Santé
+  url: https://www.dailymotion.com/video/x7stprh
+  archive: https://web.archive.org/web/20200320115928/https://www.dailymotion.com/video/x7stprh
+donneesNationales:
+  casConfirmes: 10995
+  deces: 372
+  casHospitalises: 4461
+  casReanimation: 1122
+  casGueris: 1300
+  CapaciteLitsDisponibles: 3700
```

Comment:
Est ce qu'il est possible de rajouter un commentaire indiquant que c'est la "somme" à peu près des 3 lignes dessous ?

Reply:
oui, on it.

Reply:
@cecilaki : Merci pour ta contribution. Je me permets d'ajouter qu'il faudrait - selon moi - que les clés soient en camelCase, par exemple: `capacitesLitsDisponibles` 
et ce afin d'être cohérent avec la convention de nommage des autres clés